pkg/cmd/cli/cmd/build.go

```diff
@@ -181,6 +181,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().BoolVarP(&(cfg.ForceCopy), "copy", "c", false, "Use local file system copy instead of git cloning the source url")
 	buildCmd.Flags().StringVar(&(cfg.RuntimeImage), "runtime-image", "", "Image that will be used as the base for the runtime image")
 	buildCmd.Flags().VarP(&(cfg.RuntimeArtifacts), "runtime-artifact", "a", "Specify a file or directory to be copied from the builder to the runtime image")
-
+	buildCmd.Flags().StringVarP(&(cfg.Network), "network", "", "", "Specify the default Docker Network name to be used in build process")
```

Comment:
when invoked via openshift, we set up a config struct explicitly with DockerNetworkMode set and invoke the s2i code directly (not via cli)

when invoked via the CLI today, s2i does not appear to set DockerNetworkMode to any value (it is an empty string) which I guess is handled as a default by the docker apis when we pass it in.

So the net is when s2i is being invoked via CLI, you should be free to set DockerNetworkMode (or leave it as empty string).